Makefile

```diff
@@ -36,47 +37,51 @@ default: build
 # and will only work - when this tree is found on the GOPATH.
 ifeq ($(CURDIR),$(PROJECT_DIR))
 
-ifeq ($(JUJU_MAKE_GODEPS),true)
-$(GOPATH)/bin/godeps:
-	go get github.com/rogpeppe/godeps
+ifeq ($(JUJU_MAKE_GODEPS),true)  # ?? change this to JUJU_MAKE_DEP
```

Comment:
If the ?? change happens extra changes will need to happen for ci-run and the pr jobs. (i.e. try using _GODEPS and if that fails use _DEP etc.).